packages/PKGBUILD

```diff
@@ -5,7 +5,6 @@ pkgrel="1"
 pkgdesc="Abstraction Layer for Carbonio"
 maintainer="Zextras <packages@zextras.com>"
 arch=('x86_64')
-license=("GPLv2")
```

Comment:
Please don't remove it, instead use
`license=("GPL-2.0-only")`
as suggested by the yap log.
My words are true only if the unique motivation of this PR is to avoid CI failures